apl/data-types/map-fields.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Map fields'
+description: 'This page explains what map fields are and how to query them.'
+tags: ['axiom documentation', 'documentation', 'axiom', 'map fields', 'object fields', 'data types']
+---
+
+Map fields are a special type of field that can hold a collection of nested key-value pairs within a single field. You can think of the content of a map field as a JSON object.
+
+## Benefits of map fields
```

Comment:
should also mention drawbacks - querying them uses more query gbms, there's no visibility into them from the schema (ie autocomplete will not know what properties are inside the map)